src/app/views/provisioning/product-pricing-price.html

```diff
@@ -0,0 +1,22 @@
+<span ng-hide="subscription.product_pricing.pricing_type == 'payg'">
```

Comment:
Should free plans be taken into account here too?

Reply:
sure